website/content/docs/developers/compiling/linux.md

```diff
@@ -41,9 +41,9 @@ sudo apt-get install oracle-java8-installer -y
 #### Step 5 - Install Bazel
 
 ```bash
-wget https://github.com/bazelbuild/bazel/releases/download/0.1.2/bazel-0.1.2-installer-linux-x86_64.sh
-chmod +x bazel-0.1.2-installer-linux-x86_64.sh
-./bazel-0.1.2-installer-linux-x86_64.sh --user
+wget https://github.com/bazelbuild/bazel/releases/download/0.2.3/bazel-0.2.3-installer-linux-x86_64.sh
```

Comment:
I recall having a problem when I first used {{% bazelVersion %}} that doesn't parse within the ```bash``` syntax.  Trying again just now it fails, parses as {@{@HUGOSHORTCODE-2@}@}.  I'll look into it further, but for tomorrow's update this hopefully suffices so users don't install Bazel 0.1.2?